redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1219 @@
+                            guild,
+                            ctx.message.created_at.replace(tzinfo=timezone.utc),
+                            "smute",
+                            user,
+                            author,
+                            reason,
+                            until=until,
+                            channel=None,
+                        )
+                    except RuntimeError as e:
+                        log.error(_("Error creating modlog case"), exc_info=e)
+            if not success_list and issue:
+                message = _(
+                    "{users} could not be muted in some channels. "
+                    "Would you like to see which channels and why?"
+                ).format(users=humanize_list([f"{u}" for u in users]))
+                return await self.handle_issues(ctx, message, issue)
+            if ctx.guild.id not in self._server_mutes:
+                self._server_mutes[ctx.guild.id] = {}
+            verb = _("has")
+            if len(success_list) > 1:
+                verb = _("have")
+            await ctx.send(
+                _("{users} {verb} been muted in this server{time}.").format(
+                    users=humanize_list([f"{u}" for u in success_list]), verb=verb, time=time
+                )
+            )
+            if issue:
+                message = _(
+                    "{users} could not be muted in some channels. "
+                    "Would you like to see which channels and why?"
+                ).format(users=humanize_list([f"{u}" for u in users]))
+                await self.handle_issues(ctx, message, issue)
+
+    async def handle_issues(self, ctx: commands.Context, message: str, issue: str) -> None:
+        can_react = ctx.channel.permissions_for(ctx.me).add_reactions
+        if not can_react:
+            message += " (y/n)"
+        query: discord.Message = await ctx.send(message)
+        if can_react:
+            # noinspection PyAsyncCall
+            start_adding_reactions(query, ReactionPredicate.YES_OR_NO_EMOJIS)
+            pred = ReactionPredicate.yes_or_no(query, ctx.author)
+            event = "reaction_add"
+        else:
+            pred = MessagePredicate.yes_or_no(ctx)
+            event = "message"
+        try:
+            await ctx.bot.wait_for(event, check=pred, timeout=30)
+        except asyncio.TimeoutError:
+            await query.delete()
+            return
+
+        if not pred.result:
+            if can_react:
+                await query.delete()
+            else:
+                await ctx.send(_("OK then."))
+            return
+        else:
+            if can_react:
+                with contextlib.suppress(discord.Forbidden):
+                    await query.clear_reactions()
+            resp = pagify(issue)
+            await ctx.send_interactive(resp)
+
+    @commands.command(
+        name="mutechannel", aliases=["channelmute"], usage="[users...] [time_and_reason]"
```

Comment:
Users argument is required.
```suggestion
        name="mutechannel", aliases=["channelmute"], usage="<users...> [time_and_reason]"
```